patches/api/0007-Adventure.patch

```diff
@@ -3801,27 +3826,30 @@ index 0db7fe1b9fe5621ceed3f4f046691e359f5949dd..30fce0df75494eb9b7409f08ea3d6ff8
      void setSuffix(@NotNull String suffix) throws IllegalStateException, IllegalArgumentException;
  
      /**
-@@ -90,8 +181,10 @@ public interface Team {
+@@ -90,8 +206,10 @@ public interface Team {
       *
       * @return team color, defaults to {@link ChatColor#RESET}
       * @throws IllegalStateException if this team has been unregistered
-+     * @deprecated in favour of {@link #color()}
++     * @deprecated in favour of {@link #color()} in conjunction with {@link #decoration()}
       */
      @NotNull
 +    @Deprecated // Paper
      ChatColor getColor() throws IllegalStateException;
  
      /**
-@@ -102,7 +195,9 @@ public interface Team {
+@@ -102,8 +220,11 @@ public interface Team {
       *
       * @param color new color, must be non-null. Use {@link ChatColor#RESET} for
       * no color
-+     * @deprecated in favour of {@link #color(net.kyori.adventure.text.format.NamedTextColor)}
++     * @throws IllegalStateException if this team has been unregistered
++     * @deprecated in favour of {@link #color(net.kyori.adventure.text.format.NamedTextColor)} or {@link #decoration(net.kyori.adventure.text.format.TextDecoration)}
       */
+-    void setColor(@NotNull ChatColor color);
 +    @Deprecated // Paper
-     void setColor(@NotNull ChatColor color);
++    void setColor(@NotNull ChatColor color) throws IllegalStateException; // Paper - add IllegalStateException to throws
```

Comment:
This is unnecessary

Reply:
do I keep the @throws since it can throw IllegalStateException or remove it all entirely